### DISCUSSION TITLE
create new user

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#16534] [How to]
What would be the best method to create new user on the basis of account, or vendor module? 
Or maybe there is a script ready-made for that?
BEst

**Comment by mariuszkrzaczkowski** [comment]
give an example of use

**Comment by s2b-git** [comment]
Conversion of candidates to regular employees or subcontractors. Here all of them should have access to own profile. We finding very convenient using profiling instead of separate portal. Automating this procedure will be of an advantage to those who dealing with larger number of incoming and outgoing system users. 
1. Activate certain module = create new user 
2. Changing data in certain module = update user
3. Deactivating with status = deactivate user
